src/include/txn.i

```diff
@@ -476,7 +415,7 @@ __wt_txn_op_set_timestamp(WT_SESSION_IMPL *session, WT_TXN_OP *op)
  *	Mark a WT_UPDATE object modified by the current transaction.
  */
 static inline int
-__wt_txn_modify(WT_SESSION_IMPL *session, WT_UPDATE *upd)
+__wt_txn_modify(WT_SESSION_IMPL *session, WT_UPDATE *upd, WT_TXN_OP** opp)
```

Comment:
It should be `WT_TXN_OP **opp` (i.e. the space comes after the `P` and before the `**`).